libs/unity/library/Runtime/Scripts/Media/AudioReceiver.cs

```diff
@@ -6,19 +6,7 @@
 
 namespace Microsoft.MixedReality.WebRTC.Unity
 {
-    /// <summary>
-    /// This component represents a remote audio source added as an audio track to an
-    /// existing WebRTC peer connection by a remote peer and received locally.
-    /// The audio track can optionally be displayed locally with a <see cref="VideoRenderer"/>.
-    /// </summary>
-    /// <remarks>
-    /// This component will play audio only while it is active and a remote track is associated
-    /// to the paired <see cref="WebRTC.Transceiver"/>.
-    /// </remarks>
-    /// <seealso cref="VideoRenderer"/>
-    [AddComponentMenu("MixedReality-WebRTC/Audio Receiver")]
-    [RequireComponent(typeof(UnityEngine.AudioSource))]
-    public class AudioReceiver : WorkQueue, IAudioSource, IMediaReceiver, IMediaReceiverInternal
+    public class AudioReceiver : MediaReceiver, IAudioSource
```

Comment:
Now receivers are separate again and simpler (no Update; events are invoked on the right thread by the MediaLine directly). VideoRenderer subscribes to the events, and AudioRenderer could do the same for symmetry.

Next step could be embed them in the MediaLine I guess? The transceiver direction could be controlled by a checkbox and the MediaLine UI could expose the video/audio events rather than having them in a separate component/object.

Reply:
> Next step could be embed them in the MediaLine

...except that then we cannot keep a MediaReceiver reference because it won't be properly serialized anymore. We'd have to keep a VideoReceiver and an AudioReceiver in each MediaLine and use only one. Not sure if it's going to be worth the effort, maybe let's keep this as optional improvement if we have time.

Reply:
The idea is that MediaReceiver would not be a MonoBehaviour anymore if it's just a piece of MediaLine.